Contributing.md

```diff
@@ -26,6 +27,16 @@ Also, this probably goes without saying, but piracy is strictly forbidden both o
 
 If you make any contributions to Dolphin after December 1st, 2014, you are agreeing that any code you have contributed will be licensed under the GNU GPL version 2 (or any later version).
 
+# <a name="prerequisites">Before You Start
+
+This is an emulator, that means that it runs games by emulating other hardware (in this case, the Wii and GameCube). In order to contribute, you will need to understand how these consoles work. You can learn this by:
+ - Searching on the web how they work (for example, [on WiiBrew](https://wiibrew.org/wiki/Main_Page), or [YAGCD](https://www.gc-forever.com/yagcd/)).
+ - Manually reverse engineering the Wii/GameCube console (we will not help you with this).
+We advise you to please do not: 
+ - [**Attempt to Learn or find Trade Secrets**](#trade-secrets).
+ - Break any relevant laws in your country.
+These are all **ILLEGAL**. We won't help you if you do it anyway.
```

Comment:
This is basically just restating the "Legal" section, which is right above this section. So it's not useful.